### PR TITLE
docs: ✏️ added info about seb-win install

### PR DESCRIPTION
**Pull request by EldRoos**
ref #306

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +83,24 @@ npm install -g git-cz
...
+```bash
+npx nx run chlorofyll:storybook
+npx nx run react:storybook
+npx nx run angular:storybook
+```
+
+Or if you install nx globally you can omit the npx above.
+
+```bash
+npm install -g nx
```
Comment (sebopensource) [inline README.md:103]: Easier to read and understand if the flags are at the end like this which also is more common practice:

```suggestion
npm install nx -g
```

File: README.md [hunk 2]
```diff
@@ -83,3 +83,24 @@ npm install -g git-cz
 ```
 
 To commit, just run `git-cz` in the terminal 😄
+
+
+### On SEB Windows computer
+
+Yarn support for our artifactory will be supported later on, but for now you need to logoff the VPN (right click the lock symbol in the tray).
+
+You start the 3 storybocks with thesse commands:
```
Comment (sebopensource) [inline README.md:92]: Fix typo:)

```suggestion
You start the 3 storybocks with these commands:
```

Comment (sebopensource) [inline README.md:92]: Maybe add instructions for generic use as well eg.

```
npx nx run [project]:storybook
```

As we have more storybooks besides the 3 in the instructions.

**Review by sebopensource — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -74,12 +75,46 @@ Unlike `npm`, `yarn` doesn't require the `run` command. So if you want to run `l
 yarn lint
 ```
 
+### How to contribute?
+
+- Clone the project locally from GitHub - `git clone https://github.com/sebgroup/green.git`.
```
Comment (hjalmers) [inline README.md:80]: We should probably add the fork option as well for users who don't belong to our github.com organisation. It's basically the same but they need to fork it first:)

**Review by hjalmers — APPROVED** [review]

**Pull request merged**